Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -303,9 +303,10 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             cid: channelController.cid!,
             messageId: message.id
         )
-
+        
         let actionsController = _ChatMessageActionsVC<ExtraData>()
         actionsController.messageController = messageController
+        actionsController.channelController = channelController
```

Comment:
We can't do this. LLC controllers can have just one delegate, we shouldn't be passing them around just like that :) Why not pass just the config?

Reply:
I see, I thought that it would make sense for the VC's to have controllers, and interact with them to get what they need, but putting only the config also works fine, and is actually easier 👍